src/wallet/wallet_rpc_server.cpp

```diff
@@ -3893,7 +3893,27 @@ namespace tools
     crypto::secret_key recovery_val;
     try
     {
-      recovery_val = wal->generate(wallet_file, std::move(rc.second).password(), recovery_key, true, false, false);
+      if (req.enable_multisig_experimental)
+      {
+        // Parse multisig seed into raw multisig data
+        epee::wipeable_string multisig_data;
+        multisig_data.resize(req.seed.size() / 2);
+        epee::span<std::uint8_t> ms_span{reinterpret_cast<std::uint8_t*>(multisig_data.data()), multisig_data.size()};
```

Comment:
This likely works, but I think unwrapping is a bit safer `epee::as_mut_span(unwrap(unwrap(multig_data)))`. This does some basic checks to make sure that casting to a `uint8_t*` is safe.

Reply:
`epee::to_mut_span` does not work here because the `value_type` of `wipeable_string` is `char`, not `std::uint8_t`.

Reply:
I could write a `to_mut_byte_span` function

Reply:
Sorry, I meant `as_mut_byte_span(unwrap(unwrap(foo)))`, which already exists.

Reply:
I think that function is only for POD types and not STL-like containers, though, right?

Reply:
My apologies, I was thinking of the other class which works with `unwrap(...)`.